src/app/resources/axis2/welcome.html

```diff
@@ -0,0 +1,194 @@
+    	 	<td align="right"><img src="/resources/images/axis_l.jpg" alt="AXIS Logo"></td>
+          </tr>
+        </tbody></table>
+		</td>
+		<td>
+		&nbsp;
+		</td>
+	  </tr>
+	</tbody></table>
+
+    
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<table summary="back home table" width="100%">
+	<tbody><tr><td>
+		<table summary="embedded back home table">
+			<tbody><tr>
+				<td align="right"><a href="#">Back Home</a>&nbsp;&nbsp;|&nbsp;&nbsp; <a href="#" onclick="javascript:window.location.reload(true);return false;">Refresh</a></td>
+			</tr>
+		</tbody></table>
+	</td>
+	<td>&nbsp;</td></tr>
+</tbody></table>
+
+    <table summary="main content table" class="FULL_BLANK">
+      <tbody><tr>
+        <td valign="top">
+        	<h2>Welcome :</h2>
+        	<p>Welcome to the Axis2 administration console. From inside the Axis2 administration console you can :</p>
+	    	<ul style="list-style: none;" class="loginUL">
+	       		<li>Check on the health of your Axis2 deployment.</li>
+ 	       		<li>Change any parameters at run time.</li>
+ 	       		<li>Upload new services into Axis2 [Service hot-deployment].</li>
+ 	   		</ul>
+	        <span style="color:orange">Warning: Please note that configuration changes done through the administration console
+    	    will be lost when the server is restarted.</span>
+    	</td>
+        <td valign="top" align="left">
+          <form method="post" name="LoginForm" action="/axis2-admin/login">
+            <table summary="login table" class="LOG_IN_FORM">
+              <tbody><tr>
+                <td colspan="2" style="bgcolor:#b0b0b0" align="center"><span style="color:#FFFFFF">Login</span></td>
+              </tr>
+              <tr>
+                 <td colspan="2" align="center">&nbsp;</td>
+               </tr>
+              <tr>
+                <td align="right">Username:</td>
+                <td><input style="align:left" type="text" name="userName" tabindex="1">
+                </td>
+              </tr>
+              <tr>
+                <td align="right">Password : </td>
+                <td><input style="align:left" type="password" name="password" tabindex="2">
+                </td>
+              </tr>
+              <tr>
+                <td colspan="2">
+                  <br>
+                </td>
+              </tr>
+              <tr>
+                <td colspan="2" align="center">
+                  <input name="cancel" type="reset" value=" Clear "> &nbsp; &nbsp;
+                  <input name="submit" type="submit" value=" Login ">
+                </td>
+              </tr>
+              <tr>
+                <td colspan="2" align="center">
+                  <span style="color:red">&nbsp;&nbsp;</span>
+                </td>
+              </tr>
+            </tbody></table>
+          </form>
+          <br><br><br><br><br><br>
+        </td>
+      </tr>
+    </tbody></table>   
+    <!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Do we need the same license twice in the same file? Same comment for the other file.

Reply:
Nope. A single license should be enough. Resolved.